Client/Frontend/Browser/FaviconHandler.swift

```diff
@@ -28,7 +28,7 @@ class FaviconHandler {
             return deferMaybe(FaviconError())
         }
 
-        let deferred = Deferred<Maybe<(Favicon, Data?)>>()
+        let deferred = Deferred<Maybe<(Favicon, Data?)>>(value: nil, defaultQueue: .main)
```

Comment:
why does it need to run on main queue, do you remember?

Reply:
Because it crashes if it doesn't: https://bravesoftware.slack.com/archives/GB9CJGQV6/p1575324798015800

Reply:
Because it crashes if it doesn't: https://bravesoftware.slack.com/archives/GB9CJGQV6/p1575324798015800